render.yaml

```diff
@@ -65,13 +65,13 @@ services:
     routes:
       - type: rewrite
         source: /api/
-        destination: 'https://chapp-api.sergeylukin.com/api/'
+        destination: 'https://chapp-api-pr-2.onrender.com/api/'
       - type: redirect
         source: /api
         destination: '/api/'
       - type: rewrite
         source: /api/*
-        destination: 'https://chapp-api.sergeylukin.com/api/*'
+        destination: 'https://chapp-api-pr-2.onrender.com/api/*'
```

Comment:
This is the last and only manual step required for PR previews to work at their fullest potential (by redirecting /api to instance spinned up for PR rather than production API) due to current limitation of render.com (it's not supporting `fromService` in `routes` unlike `envVars` which does work with `fromService` references).
P.S. This feature would be a great life saver for me personally.